File: shift.py
#!/usr/bin/python3

def shift():
    m = input("enter message: ")
    m = m.lower()
    k = int(input("enter key: "))
    res = ''
    opt = input("encrypt or decrypt? (e/d): ")
    if opt == 'e':
        for i in m:
            res += chr( (((ord(i) - 97) + k) % 26)+97 )
        return print('secret: ' + res)
    elif opt == 'd':
        for i in m: 
            res += chr( (((ord(i) - 97) - k) % 26) + 97)
        return print('message: ' + res)

def main():
    shift()

if __name__ == '__main__':
    main()

File: shift.c
#include <stdio.h>
#include <string.h>

int shift();

void main(){
    
    shift();

}

int shift(){

    int k,i,t;
    char m[50],res[50],opt;

    printf("enter message and key: ");
    scanf("%s %d", &m, &k);
    printf("encrypt or decrypt?(e/d): ");
    scanf(" %c", &opt);
    if(opt=='e'){
       for(i=0;i<strlen(m);i++){
           t = m[i] - 97;
           res[i] = ( (t + k)%26 ) + 97;
       } 
       printf("secret: %s", &res);
    }
    else if(opt=='d'){
        for(i=0;i<strlen(m);i++){
            t = m[i] - 97;
            res[i] = ( (t - k)%26 ) + 97;
        }
        printf("message: %s", &res);
    }
    return 0;
}
